### DISCUSSION TITLE
Organize contacts by labels

### DISCUSSION THREAD

**Discussion by lahrim** [tutao/tutanota#5534] [Ideas]
Be able to organize contacts by label as for emails with folders

**Comment by jcmuel** [comment]
[Contact labels / groups](https://support.google.com/contacts/answer/30970) are a standard functionality of the Contacts app of the Android Open Source Project (AOSP).

It is currently not possible to use the AOSP Contacts app to create a contact label in the Tuta account.

Steps to reproduce:
- On an AOSP device, open the Contacts app
- Open the menu
- Tap on "Create label"
- A popup opens with the title "Choose account"
- Observe that the Tuta account is missing in the list. 
  - Only the local device account, Google accounts, and DAVx5 accounts are listed.
